filebeat/docs/modules/azure.asciidoc

```diff
@@ -119,6 +119,16 @@ https://management.azure.com/ for azure PublicCloud
 https://management.usgovcloudapi.net/ for azure USGovernmentCloud
 Users can also use this in case of a Hybrid Cloud model, where one may define their own endpoints.
 
+`resource_manager_audience` ::
+_string_
+Optional, by default we are using the azure public environment, to override, users can provide a specific resource manager audience in order to use a different azure environment.
+Ex:
+https://management.chinacloudapi.cn/ for azure ChinaCloud
+https://management.microsoftazure.de/ for azure GermanCloud
+https://management.azure.com/ for azure PublicCloud
+https://management.usgovcloudapi.net/ for azure USGovernmentCloud
+Users can also use this in case of a Hybrid Cloud model, where one may define their own audiences.
+
 include::../include/what-happens.asciidoc[]
```

Comment:
Hey @dedemorton @colleenmcginnis, this PR adds a new optional setting named `resource_manager_audience` to Metricbeat. 

My understanding is that by updating this .asciidoc file, we are updating the public docs at:

https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-azure.html#_module_specific_configuration_notes_2 

I have a couple of questions:

- Is this accurate?
- Are there other docs we need to take care of by adding this configuration option?

Thanks!

Reply:
Here's your clue for the build error:

```
Error: some files are not up-to-date. Run 'make update' then review and commit the changes. Modified: [filebeat/docs/modules/azure.asciidoc]
```

Reply:
> It looks like you updated the wrong file. 

Oh my, thanks for the heads up! I made an n00b error.. 🤦‍♂️  I am reverting the previous change and updating the correct file.